gobblin-core/src/main/java/org/apache/gobblin/writer/FsDataWriter.java

```diff
@@ -266,7 +267,7 @@ public void commit()
     LOG.info(String.format("Moving data from %s to %s", this.stagingFile, this.outputFile));
     // For the same reason as deleting the staging file if it already exists, overwrite
     // the output file if it already exists to prevent task retry from being blocked.
-    HadoopUtils.renamePath(this.fs, this.stagingFile, this.outputFile, true);
+    HadoopUtils.renamePath(this.fs, this.stagingFile, this.outputFile, true, this.conf);
```

Comment:
I would rather call renamePath without conf object which will take care of the calling the other methods with new conf object